controllers/account.go

```diff
@@ -175,14 +176,38 @@ func (c *ApiController) addInitialChatAndMessage(user *casdoorsdk.User) error {
 		return err
 	}
 
+	store, err := object.GetStore(util.GetId("admin", chat.Store))
+	if err != nil {
+		return err
+	}
+
+	userMessage := &object.Message{
+		Owner:        "admin",
+		Name:         fmt.Sprintf("message_%s", util.GetRandomName()),
+		CreatedTime:  chat.CreatedTime,
+		Organization: chat.Organization,
+		User:         userName,
+		Chat:         chat.Name,
+		ReplyTo:      "",
+		Author:       userName,
+		Text:         store.Welcome,
+		IsHidden:     true,
+		VectorScores: []object.VectorScore{},
+	}
+	_, err = object.AddMessage(userMessage)
+	if err != nil {
+		return err
+	}
+	time.Sleep(1000 * time.Millisecond)
+
 	answerMessage := &object.Message{
 		Owner:        "admin",
 		Name:         fmt.Sprintf("message_%s", util.GetRandomName()),
 		CreatedTime:  util.GetCurrentTimeEx(chat.CreatedTime),
 		Organization: chat.Organization,
```

Comment:
You used hidden message to replace "Welcome", I didn't see your AI request logic modification